src/Parser/DisciplinePriest/Modules/Spells/Penance.js

```diff
@@ -0,0 +1,66 @@
+import SPELLS from 'common/SPELLS';
+
+import Module from 'Parser/Core/Module';
+
+/* This module will try to use Speed of the Pious to determine new Penance casts
+ * If Speed of the Pious is not available, we fallback to an estimate
+ */
+
+const PENANCE_MINIMUM_RECAST_TIME = 3500; // Minimum duration from one Penance to Another
+
+class Penance extends Module {
+  priority = 99;
```

Comment:
A priority of `9` would better fit this module. Do a find in files for `priority =` to find similar module priorities.